maven-plugin/src/main/java/org/jvnet/localizer/Generator.java

```diff
@@ -111,7 +171,28 @@ protected void generateImpl(String className, Properties props) throws Assertion
         }
     }
 
+    private String code(String value) {
+        return String.format("{@code %s}", escape(value));
+    }
+
     private String escape(String value) {
-        return value.replace("&","&amp;").replace("<","&lt;");
+        return value.replace("&","&amp;").replace("<","&lt;").replace(">", "&gt;");
     }
+
+    private String positionalName(int i) {
+        if (i >= 9 && i < 20) {
+            return String.format("%dth", i+1);
+        }
```

Comment:
11th, 12th, 13th but after that it's 21st 22nd 23rd, etc... granted I should probably be doing everything modulo 100... but if we have 100 parameters something is seriously wrong